src/components/Note/Note.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./Note.css";
+
+function Note({ title, note, color }) {
+  return (
+    <div class={`card m-1 p-2 note-color-${color}`}>
```

Comment:
rename to className

Reply:
Thanks for pointing it out. I have removed it in next pr